src/main/java/place_word_refill_user_story/PlaceWordRefillController.java

```diff
@@ -0,0 +1,5 @@
+package place_word_refill_user_story;
+
+public interface PlaceWordRefillController {
+    // TODO: What even goes here?
```

Comment:
I believe this is something we can address with whoever is writing the code for the bag class